chapters/install/ide-speedup.md

```diff
@@ -0,0 +1,90 @@
+# Speed-up Jina App development in PyCharm and VSCode
+
+Jina App development can be time consuming if you donot have the IDE setup correctly. This chapter describes how to speed up your developer experience
+in Pycharm and VS Code IDE while using Jina. In particular we would focus on how to enable Intellisense for Jina YAML configs. Intellisense can provide code completion, showing number and arguments list, validating your YAML config.
+
+These features can help you build your Jina application much faster.
+
+## Set up your developer environment for jina
+
+Once you have the latest Jina version installed with PyPI, while creating a typical Jina application - two types of files need to be written: Python code and YAML config. The Python file defines the entry point and customized logics while the YAML configs define the flow composition and the configuration of each individual Executor. Depending on the cases and the depth of customization, you can develop the project in both ways. In Jina, it is even possible to build a completely code free project that only depends on the configs. Most IDEs have provided smart intelligence for Python development - due to
```

Comment:
The first sentence is too long, and you cross tenses. You go from the past to the present. 

Once you have the latest Jina version installed with PyPI. You can begin to create your Jina application, two types of files need to be written: Python code and YAML config.

Reply:
logic not logics